src/tools/dartpad.md

```diff
@@ -168,3 +168,9 @@ To create a simple web app, start with the Hello World HTML sample.
     </ol>
   </li>
 </ol>
+
+### Checking DartPad version info
+
+To check what version of DartPad you're running, 
```

Comment:
Is there a possibility to change you DartPad version to anything later or maybe even to snapshots? 
If so, maybe include a _how to change your DartPad version_ section aswell?  At the moment i found only 1.25 and Dart 2 so i guess this would be irrelevant.